### ISSUE TITLE
isatab.validate() doesn't print logging info

### ISSUE THREAD

**Issue by hcwi** [ISA-tools/isa-api#259]
The function validate() won't produce any logging info; it seems to ignore the argument log_level=config.log_level

**Comment by hcwi** [comment]
https://github.com/ISA-tools/isa-api/blob/49c36f8d180057832a1ab0711dacff07e3d04926/isatools/isatab.py#L2487

`log.setLevel(log_level)` <- no effect
`log.setLevel(config.log_level)` <- unresolved reference to config error unless imported again (then no effect)
`log.setLevel(logging.INFO)` <- ok

**Comment by djcomlab** [comment]
Hi @hcwi 

Not sure what the problem is here. If you do:
```
from isatools import isatab
import logging
isatab.validate(open('BII-I-1/i_investigation.txt'), log_level=logging.INFO)
```
the console should output `INFO`, `WARNING` and `ERROR` messages (in the BII-I-1 case there's only info and warning's logged). Likewise if you just set to `logging.WARNING` but will only show warning messages.

Please can you provide an example of the code you are trying that doesn't work?

**Comment by hcwi** [comment]
Hi @djcomlab,

I'm sorry for the vague description. I have now checked again more carefully, and what seemed to be the problem might be just my misunderstanding, i.e.:

I have tried to set the default logging level through the _config.py_: 
https://github.com/ISA-tools/isa-api/blob/49c36f8d180057832a1ab0711dacff07e3d04926/isatools/config.py#L6
but without success (no log messages). Setting `log_level` in the code directly, without referencing the _config.py_, worked fine.
It seems that the `read()` function in config.py overrides the `log-level` parameter with values from _isatools.ini_ file (which in this case was ERROR and thus resulted in no messages for the datasets tested, which appeared as non working).

To sum up, setting the `log-level` parameter in the _config.py_ is ineffective, the default logging level should be declared in _isatools.ini_, and the problem was that I didn't expect it.

**Comment by djcomlab** [comment]
I'll check this further - actually if you do `config.read('my.ini')` it's supposed to set the global logging level manually (if not already overridden in the calls to the other functions) rather than changing the log level inside the `config.py` code, but maybe this doesn't work...

The idea is to give user control of the global log level without modifying any code

**Comment by djcomlab** [comment]
Actually on further checks the `config.read()` doesn't seem to work as expected in changing the log level this way, so needs fixing.

**Comment by djcomlab** [comment]
Fixed in 22a209c
